x/observer/types/params.go

```diff
@@ -52,9 +56,11 @@ func DefaultParams() Params {
 			PolicyType: Policy_Type_update_keygen_block,
 			Address:    "zeta1afk9zr2hn2jsac63h4hm60vl9z3e5u69gndzf7c99cqge3vzwjzsxn0x73",
 		},
+		{
+			PolicyType: Policy_Type_add_observer,
+			Address:    "zeta1afk9zr2hn2jsac63h4hm60vl9z3e5u69gndzf7c99cqge3vzwjzsxn0x73",
```

Comment:
We're reusing the same hardcoded address many places, I think we should define a constant